app/screens/image_preview/downloader.ios.js

```diff
@@ -423,7 +427,8 @@ const styles = StyleSheet.create({
         width: 200,
         height: 200,
         alignItems: 'center',
-        justifyContent: 'center',
+        justifyContent: 'flex-start',
+        paddingTop: 40,
```

Comment:
Happy to pull this UI change into a separate PR and/or create a separate GH issue. I couldn't help myself.

DownloaderBottomContent conditionally renders inside a fixed width and height container whose contents are vertically centered. Unfortunately this causes the circular progress component to noticeably shift up to make room for the DownloaderBottomContent text.

Since the container has a fixed width and height:  flex-start + padding for the circular progress component keeps everything vertically centered and leaves room for DownloaderBottomContent text to come and go without the rest of the container's children shifting around.

Here's what it looked like before this change with Slow Animations:

![download_image_jank](https://user-images.githubusercontent.com/1521460/69122479-cad8f180-0a53-11ea-9064-317636074f05.gif)

The after can be seen in the gif in the summary.

Reply:
As an aside there are two polish issues that would help make this component shine:

1) The circular progress bar never has a chance to show 100% - ideally the progress bar would show 100% for at least 250ms before the downloader is hidden.
2) The downloader animates in from the bottom but doesn't animate away. A matching dismiss animation would be nice.

Do we want to create a new issue for those two things?

Reply:
Yeah, that would be nice to have in a separate issue

Reply:
@smacgregor feel free to submit a follow-up PR for the above and we'll get a ticket created then